manageiq-providers-red_hat_virtualization.gemspec

```
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "ovirt-engine-sdk", "~>4.4.0"
+  spec.add_dependency "ovirt-engine-sdk", "~>4.6.0"
   spec.add_dependency "ovirt_metrics", "~>3.2"
```

As discussed in the [other thread](https://github.com/ManageIQ/manageiq-providers-ovirt/pull/676#issuecomment-2342019635), I wonder if it's cleaner / better to do the following

```suggestion
  spec.add_dependency "manageiq-providers-ovirt"
```